src/toil/jobStores/aws/jobStore.py

```diff
@@ -47,6 +49,47 @@
 
 log = logging.getLogger(__name__)
 
+s3partSize = 2**20*50 
+
+
+def copyKeyMultipart(newKeyName, dstBucketName, srcBucketName, srcKey, headers={}):
+    """
+    Copies a key pointed at by the source URL to the destination URL in multiple parts.
+
+    :param str newKeyName: The name of the new key that will be created.
```

Comment:
You don't know if this is a new key. It could be an existing key that you are overwriting.